### PR TITLE
Autoscaling in Amazon Web Services (AWS): An Overview

### LINKED ISSUE #626: Autoscaling in Amazon Web Services (AWS): An Overview
### Brief Summary: 
Amazon Web Services (AWS) are cloud computing services that provide Application Programming Interfaces (APIs) and platforms for cloud computing to governments, individuals and companies. This article aims at providing readers with a broad understanding of autoscaling in AWS, which is an important concept for developers.

### Key Takeaways: 
The following are important concepts that readers should understand;
1. What is autoscaling? 
2. How AWS autoscaling works
3. Types of AWS autoscaling
4. Components involved in AWS autoscaling
5. Benefits of AWS Autoscaling
6. Conclusion

### References: 
https://www.whizlabs.com/blog/amazon-web-services/#:~:text=Introduction%20to%20Amazon%20Web%20Services%20Amazon%20Web%20Services,a%20private%20scalable%20infrastructure%20and%20network%20for%20themselves.

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #626 . This PR has been created because the previous PR has a merge conflict.

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by cyberShaw** [comment]
I'll do a review by EOD today. @mbaabuones

**Comment by hectorkambow** [comment]
@cyberShaw 
Just trying to get a status update on this article 🚀

File: articles/autoscaling-in-aws/index.md
Comment (cyberShaw) [inline articles/autoscaling-in-aws/index.md:94]: Really good article, @mbaabuones!
I have one suggestion, could you please add a section where you give a mini-tutorial on how to set up auto scaling on an instance. 
Your article provided very good insight on Autoscaling, but I felt the article would provide more value if you give some hands-on to the topic you're discussing about.

**Review by cyberShaw — CHANGES_REQUESTED** [review]
Great article, @mbaabuones!

There were very few things to change, so I combined the tech and the initial review into one. Please make the suggested changes and we can get this ready for final review.

**Comment by mbaabuones** [comment]
@cyberShaw Thank you for your suggestions. I have finished working on all the changes.

**Comment by cyberShaw** [comment]
> @cyberShaw Thank you for your suggestions. I have finished working on all the changes.

Amazing work, I checked out the changes and they look really good.
This article is polished and is ready for final review. 🔥

**Comment by cyberShaw** [comment]
@mbaabuones Can you add a hero image for this article?

**Pull request merged**
